src/content/deployment/_index.en.md

```diff
@@ -5,3 +5,58 @@ weight: 10
 pre: "<b>2. </b>"
 ---
 
+The inner details of deploying a broker and connecting clusters to the broker are complicated, subctl automates and simplifies most of those details eliminating human error as much as possible. This is why **subctl** is the recommended deployment method, you can find a complete guide to the subctl tool here: [subctl in detail](subctl). If you still believe _helm_ works better for you, please go [here](helm).
+
+## Installing subctl
+
+Download the subctl binary and make it available on your PATH.
+
+<!-- TODO: create a "getsubctl.sh" to simplify this -->
+```bash
+mkdir -p ~/.local/bin
+curl -L -o ~/.local/bin/subctl https://github.com/submariner-io/submariner-operator/releases/download/v0.1.0/subctl-v0.1.0-linux-amd64
+chmod a+x ~/.local/bin/subctl
+export PATH=$PATH:~/.local/bin
+echo export PATH=\$PATH:~/.local/bin >> ~/.profile
+```
+
+
+## Deployment of broker
+
+Please remember, this cluster's API should be accessible from all other clusters:
+```bash
+subctl deploy-broker --kubeconfig <PATH-TO-KUBECONFIG-BROKER> 
+```
+
+this will create:
+
+* The submariner-k8s-broker namespace
+* The Cluster and Endpoint (.submariner.io) CRDs in the cluster
+* A service account in the namespace for subsequent subctl access.
+* A random IPSEC PSK which will be stored only on the broker-info.subm file.
+
+And generate a broker-info.subm file which contains the following elements:
+
+* The API endpoint
+* A CA certificate to for the API endpoint
+* The service account token for accessing the API endpoint / submariner-k8s-broker namespace.
+
+
+{{% notice info %}}
+This cluster can also participate on the dataplane connectivity with the other clusters, but it will need to be joined (see following step)
```

Comment:
```suggestion
This cluster can also participate in the dataplane connectivity with the other clusters, but it will need to be joined (see following step)
```